state-chain/chains/src/ccm_checker.rs

```diff
@@ -93,6 +93,13 @@ impl DecodedCcmAdditionalData {
 			_ => None,
 		}
 	}
+
+	pub fn address_lookup_tables(&self) -> Option<Vec<SolAddress>> {
+		match self {
+			DecodedCcmAdditionalData::Solana(sol_data) => sol_data.address_lookup_tables(),
```

Comment:
I don't think we need this method - it can be a bit misleading since a return value of `None` can indicate the wrong chain (not Solana) or valid but no lookup tables. Better to just inline this.